src/index.ts

```diff
@@ -7,4 +7,5 @@ export * from './components/skeleton'
 export * from './components/table'
 export * from './components/tabs'
 export * from './components/typography'
+export * from './components/badge'
```

Comment:
Not too important also, but might be nice to keep the exports alphabetical order here :)